app/model.py

```diff
@@ -22,11 +22,15 @@
 from google.appengine.api import datastore_errors
 from google.appengine.api import memcache
 from google.appengine.ext import db
+from google.appengine.api import search
 
 import config
 import indexing
 import pfif
 import prefix
+import re
```

Comment:
Separate imports into Python standard modules and Person Finder's own modules, and then sort each of them in alphabetical order.